app/livechat/server/lib/Livechat.js

```diff
@@ -495,6 +496,7 @@ export const Livechat = {
 		}
 
 		Meteor.defer(() => {
+			Apps.getBridges().getListenerBridge().livechatEvent(AppEvents.IPostLivechatRoomSaved, roomData._id);
```

Comment:
Prefer the shorter wrapper method

```suggestion
			Apps.triggerEvent(AppEvents.IPostLivechatRoomSaved, roomData._id);
```